changes.go

```diff
@@ -0,0 +1,37 @@
+package sgreplicate
+
+type ChangedRev struct {
+	Revision string `json:"rev"`
+}
+
+type Change struct {
+	Sequence    interface{}  `json:"seq"`
+	Id          string       `json:"id"`
+	ChangedRevs []ChangedRev `json:"changes"`
+	Removed     []string     `json:"removed"`
```

Comment:
It's a little hard to see the diff since I moved some structs from rest_data.go -> changes.go, but the `Removed` field was added here as part of this change set.